code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -176,6 +193,13 @@
 	. = ..()
 	if(.)
 		weapon_weight = WEAPON_MEDIUM
+		inhand_icon_state = "gun"
+		worn_icon_state = "gun"
```

Comment:
this should already be handled by the base proc

Reply:
Reverted. Updated the base sawoff proc to have the worn_icon_state set to "gun". Adding the inhands to the list of missing sprites.